src/components/ModelTableList/shared.js

```diff
@@ -125,3 +125,21 @@ export function getStatusValue(status, key) {
   }
   return returnValue;
 }
+
+/**
+  Renders the controller UUID element. If the UUID is the JAAS controller then
+  it replaces the UUID with "JAAS".
+  @param {string} uuid The controller UUID.
+  @returns {Object} The element to render the controller UUID entry
+*/
+export function generateControllerUUID(uuid) {
+  let controllerUUID = uuid.split("-")[0] + "...";
+  if (uuid === "a030379a-940f-4760-8fcf-3062b41a04e7") {
+    controllerUUID = "JAAS";
+  }
+  return (
+    <a href="#_" className="p-link--soft" title={controllerUUID}>
```

Comment:
I think we should remove the link until we have the functionality.

Reply:
Sure, done.